.gitlab-ci-black-format-code.yml

```diff
@@ -0,0 +1,73 @@
+# .gitlab-ci-black-format-code.yml
+#
+# This gitlab-ci file runs the Black code formatter https://github.com/psf/black
+# on the Python3 code in the repository, and fails the pipeline if the code is
+# not formatted according to standard Black formatting rules.
+#
+# Add it to the project in hand through Gitlab's include functionality
+# 
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-black-format-code.yml'
+#
+#
+# Some things to consider before including this file. The Black code formatter
+# accepts a project-local configuration file
+# (https://github.com/psf/black#pyprojecttoml). This enables project specific
+# configuration of the Black tool.
+#
+# The recommended parameters to specify are:
+# * The Python version
+# * Include and exlude files
+#
+#
+# Example 'pyproject.toml' file:
+#
+#[tool.black]
+# target-version = ['py38']
+# include = '\.pyi?$' # Regular expression
+# exclude = '''       # Regular expression
+# (
+#   /(
+#       \.eggs         # exclude a few common directories in the
+#     | \.git          # root of the project
+#     | \.hg
+#     | \.mypy_cache
+#     | \.tox
+#     | \.venv
+#     | _build
+#     | buck-out
+#     | build
+#     | dist
+#   )/
+#   | foo.py           # also separately exclude a file named foo.py in
+#                      # the root of the project
+# )
+# '''
+# The tools also ignores all files found in .gitignore.
```

Comment:
But not `.git` folder?

Reply:
Nope, it does seem to keep away from the git folder automatically.